### PR TITLE
Added | Fairyanime

### PR DESCRIPTION
**Pull request by Yusaki-Akito**
On browsing
image

On anime select 
image

On watching
image
image

Note: The logo is too big so I have cut it down and go like that I don't have the other logo for this website

### CODE REVIEW

**Comment by Yusaki-Akito** [comment]
Fixed on SnowBoltz
Docs head change
image

File: websites/F/FairyAnime/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "en": "FairyAnime stream your favorite anime completely for free with thai-sub and thai sound!",
+    "th": "FairyAnime เว็ปไซต์สตรีมมิ่งอนิเมะที่ทุกคนชื่นชอบ ฟรีเเละมีซับไทย"
+  },
+  "url": "fairyanime.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/VHH4EsR.png",
+  "thumbnail": "https://i.imgur.com/QvrAmhs.png",
+  "color": "#f7a1f3",
+  "tags": [
+    "FairyAnime",
```
Comment (Bas950) [inline websites/F/FairyAnime/dist/metadata.json:17]: no capital letters in tags

File: websites/F/FairyAnime/presence.ts [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    }
+  }
+
+  if (presenceData.details == null) {
+    presence.setTrayTitle();
+    presence.setActivity();
+  } else {
+    presence.setActivity(presenceData);
+    //console.log(presenceData);
+  }
```
Comment (Bas950) [inline websites/F/FairyAnime/presence.ts:114]: remove that comment

File: websites/F/FairyAnime/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "en": "FairyAnime stream your favorite anime completely for free with thai-sub and thai sound!",
+    "th": "FairyAnime เว็ปไซต์สตรีมมิ่งอนิเมะที่ทุกคนชื่นชอบ ฟรีเเละมีซับไทย"
+  },
+  "url": "fairyanime.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/VHH4EsR.png",
+  "thumbnail": "https://i.imgur.com/QvrAmhs.png",
+  "color": "#f7a1f3",
+  "tags": [
+    "fairyAnime",
```
Comment (Bas950) [inline websites/F/FairyAnime/dist/metadata.json:17]: a is still in cap

**Comment by frozaum** [comment]
gud stuff, but before I approve this, a simple little tip for you, you can resize the 'FairyAnime' logo to fit the 512x (over your discord application, not premid metadata) so it looks much better than being weird

**Pull request merged**